.expeditor/scripts/bump_version.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# This script automates the modification of our Homebrew tap for the
+# `hab` binary.
+
+set -euo pipefail
+
+if ! sed --version 2>&1 | grep -q "GNU sed"; then
+    echo "This script requires GNU sed; aborting"
+    exit 1
+fi
+
+environment_name="${1:-stable}"
+s3_prefix="http://${s3_bucket_name}.s3.amazonaws.com/${environment_name}/latest/habitat/"
```

Comment:
Any reason to not hard-code `stable` here?

For instance, it's hard-coded on line 42 below.

Reply:
Also, I don't think `s3_bucket_name` is defined. Might as well hard-code it, too.